sdk/keyvault/Azure.Security.KeyVault.Secrets/src/Attributes.cs

```diff
@@ -0,0 +1 @@
+﻿[assembly: Azure.Core.AzureSdkClientLibraryAttribute("keyvault-secret")]
```

Comment:
Are we sure this is the right value?

Reply:
No, I don't know what we want this string to be.  We can change this once we have guidance.